docs/docs/partials/_InitializeDgProject.md

```diff
@@ -0,0 +1,30 @@
+import InstallUv from '@site/docs/partials/\_InstallUv.md';
+
+<Tabs groupId="package-manager">
+    <TabItem value="uv" label="uv">
+        Ensure you have `dg` [installed globally](/guides/labs/dg) as a `uv` tool:
+
+        <CliInvocationExample contents="brew install uv && uv tool install dagster-dg" />
+
+        Now run the below command. Say yes to the prompt to run `uv sync` after scaffolding:
+
+        <CliInvocationExample path="docs_snippets/docs_snippets/guides/components/index/2-a-uv-scaffold.txt" />
+
+        The `dg init` command builds a project at `jaffle-platform`. Running `uv sync` after scaffolding creates a virtual environment and installs the dependencies listed in `pyproject.toml`, along with `jaffle-platform` itself as an [editable install](https://setuptools.pypa.io/en/latest/userguide/development_mode.html). Now let's enter the directory and activate the virtual environment:
```

Comment:
Note after first sentence that "you will be prompted to run uv sync"

Reply:
This is already noted on the line before the `CliInvocationExample`: "Now run the below command. Say yes to the prompt to run `uv sync` after scaffolding."